### ISSUE TITLE
Version 0.9.0?

### ISSUE THREAD

**Issue by rocket-turtle** [guilleiguaran/fakeredis#258]
Hey @guilleiguaran 
thank you for the Time you spent working on this gem.

Would it be possible for you to release a new Version?

We are stuck with redis 4.1 because there is no Version with this fix:
https://github.com/guilleiguaran/fakeredis/pull/247/files

Maybe it would be good to add these two MRs as well to stay in sync with redis:
https://github.com/guilleiguaran/fakeredis/pull/256/files
https://github.com/guilleiguaran/fakeredis/pull/257/files

Thank you.

**Comment by nevinera** [comment]
We're stuck behind this too - currently trying to decide whether to fork, monkey-patch, or switch libraries :-\

**Comment by MrHubble** [comment]
It may not apply to your scenario, but I was able to take advantage of [the EXISTS command merged fix](https://github.com/guilleiguaran/fakeredis/pull/247/files) by using the master branch in my gemfile:

```rb
gem 'fakeredis', '~> 0.8.0', github: "guilleiguaran/fakeredis", branch: 'master'
```

Hope that helps.

**Comment by rocket-turtle** [comment]
Hi @guilleiguaran,

is there anything we can do to get this moved forward?

Thank you.
Jonas

**Comment by rocket-turtle** [comment]
@nevinera What was your Solution?

**Comment by nevinera** [comment]
> @nevinera What was your Solution?

Deferred and crossed my fingers -.-
We'll probably make a private fork if the changes still aren't released when I get back around to it.

**Comment by guilleiguaran** [comment]
v0.9.0 is published and compatible with ~> 4.8, v1.0.0 will support ~> 5.0
